infra/binder/src/main/java/org/apache/shardingsphere/infra/binder/segment/select/projection/engine/ProjectionsContextEngine.java

```diff
@@ -115,32 +114,37 @@ private boolean containsProjection(final OrderByItemSegment orderByItem, final C
     }
     
     private boolean isSameColumn(final Projection projection, final ColumnSegment columnSegment) {
-        Collection<ColumnProjection> columns = getColumnProjections(projection);
+        Collection<Projection> columns = expandProjection(projection);
         if (columns.isEmpty()) {
             return false;
         }
         boolean columnSegmentPresent = columnSegment.getOwner().isPresent();
-        for (ColumnProjection each : columns) {
+        for (Projection each : columns) {
             if (columnSegmentPresent ? isSameQualifiedName(each, columnSegment.getQualifiedName()) : isSameName(each, columnSegment.getQualifiedName())) {
                 return true;
             }
         }
         return false;
     }
     
-    private Collection<ColumnProjection> getColumnProjections(final Projection projection) {
-        Collection<ColumnProjection> result = new LinkedList<>();
-        if (projection instanceof ColumnProjection) {
-            result.add((ColumnProjection) projection);
-        }
+    /**
+     * expand projection, such as ShorthandProjection.
```

Comment:
Please keep the first letter uppercase in javadoc, and add a new line between javadoc and param doc.